device/rsvp/rsvpFastReroute.yaml

```diff
@@ -0,0 +1,75 @@
+components:
+  schemas:
+    Rsvp.FastReroute:
+      description: >-
+        Configuration for the optional RSVP-TE FAST_REROUTE object included in Path Messages as defined in RFC4090. 
+      type: object
+      properties:
+        setup_priority:
+          description: >-
+            Specifies the value of the Setup Priority field. This controls whether the backup LSP should pre-empt existing 
+            LSP setup with certain Holding Priority if resource limitation is encountered when setting up the LSP. (e.g bandwidth availability)
```

Comment:
Specifies the value of the Setup Priority field. This controls whether the backup LSP should pre-empt existing 
            LSP that is setup with certain Holding Priority. While setting up a backup LSP, preemption of existing LSP can happen if resource limitation is encountered  (e.g bandwidth availability)

Reply:
Done.